src/uct/ib/mlx5/dv/ib_mlx5_dv.h

```diff
@@ -33,6 +33,26 @@ typedef struct {
  */
 ucs_status_t uct_ib_mlx5dv_init_obj(uct_ib_mlx5dv_t *obj, uint64_t type);
 
+#if HAVE_DC_DV
+/**
+ * Initialize DC-specific DV QP attributes.
+ */
+void uct_ib_mlx5dv_dc_qp_init_attr(struct mlx5dv_qp_init_attr *dv_attr,
+                                enum mlx5dv_dc_type dc_type);
+
+/**
+ * Update QP attributes for DCI.
+ */
+void uct_ib_mlx5_dci_qp_update_attr(uct_ib_qp_init_attr_t *qp_attr);
+#endif
```

Comment:
should not `endif` be below, after `uct_ib_mlx5_dct_qp_init_attr`?

Reply:
done